views/migration/teradata/sys_calendar.sql

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* Emulation for Teradata's SYS_CALENDAR.CALENDAR table */
+CREATE OR REPLACE VIEW SYS_CALENDAR.CALENDAR (
+    /* the date */
+    calendar_date,
+    /* 1-7 day of week */
+    day_of_week,
+    /* 1-n day of month */
+    day_of_month,
+    /* 1-365 day of year */
+    day_of_year,
+    /* 1-n day since the beginning of the calendar */
+    day_of_calendar,
+    /* 1 if the date is a Monday and the first Monday of the month, etc. */
+    weekday_of_month,
```

Comment:
Could you please explain a bit more what `weekday_of_month` is maybe add more detail to the description?

Reply:
I've added a bit more to the comment here and in the README.